management-core/src/main/java/com/ns/task/service/ProductCoreServiceImpl.java

```diff
@@ -17,18 +17,27 @@
 
 @Service
 public class ProductCoreServiceImpl implements ProductService {
-
-    private final ModelMapper mapper;
-    private final ProductRepository repository;
+    private ModelMapper mapper;
+    private ProductRepository repository;
     private static final Logger logger = LogManager.getLogger();
 
+    public ProductCoreServiceImpl() {
+    }
 
     @Autowired
     public ProductCoreServiceImpl(ModelMapper mapper, ProductRepository repository) {
         this.mapper = mapper;
         this.repository = repository;
     }
 
+    @RabbitListener(queues = "q.management.get")
```

Comment:
Use the constant string defined in the configuration class

Reply:
Fixed, but I have some doubt if I can externalize this one, because  need a constant type as same as the messages for validation, so I fix here but how I have the spring cloud integration in another branch maybe it will change this ono